src/ProjectTemplates/Web.ProjectTemplates/content/RazorClassLibrary-CSharp/.template.config/template.json

```diff
@@ -20,6 +20,16 @@
   },
   "sourceName": "Company.RazorClassLibrary1",
   "preferNameDirectory": true,
+  "sources": [
+    {
+      "modifiers": [{
+        "condition": "(NoPagesAndViews)",
+        "exclude": [
+          "Areas/**"
```

Comment:
It's good that we're excluding the `.cshtml` files for the components case.

But since the other project templates all include at least one example file, should the components class library project include a `Component1.razor` file instead of being entirely empty? The content could be pretty simple, e.g.,

```html
<div>Component1</div>

@functions {
}
```

I guess it would make sense for the content to be the same as whatever we anticipate putting in the Razor Ccomponent item template, when we eventually get round to creating that. @danroth27, do you have a view on this?